api-documentation/invoice-line.md

```diff
@@ -161,3 +161,83 @@ Invoice not found.
 * `paypal`
 * `external`
 {% endhint %}
+
+## Line types
+
+The invoice lines can be of 2 basic types: increasing the invoice balance (regular lines, fees, chargebacks) and will have a positive amount attribute or decreasing the invoice balance (payments, credits) and will have a negative amount attribute. Furthermore lines can be categorized by their type (e.g.: Installment fees, Late payment fees).
+
+The type of an invoice line is determined by the `type` attributes (always present) and the optional `subtype` attributes. Below is a list of possible values.
+
+
+### Regular Lines
+
+This line type is used for lines added to the invoice at creation time, when increasing the invoice amount
+from the ClubCollect web interface or when a member adds a donation to the invoice (if enabled).
+
+| Type | Subtype | Description |
+|-|-|-|
+| `INVOICE-LINE` | | Regular Invoice line |
```

Comment:
Partners don't know what "regular" invoice lines are. Let's try to explain it in their language (and in the same line as in other pages of the API docs)

> Amounts added by the partner when an invoice is created will be of this type

Reply:
```suggestion
| `INVOICE-LINE` | | Amounts added by the partner when an invoice is created will be of this type |
```